core/domain/draft_upgrade_services.py

```diff
@@ -129,7 +129,10 @@ def _convert_html_in_draft_change_list(
             list(ExplorationChange). The converted draft_change_list.
         """
         for i, change in enumerate(draft_change_list):
-            new_value = None
+            # Here, we are using Any type because new_value can contain values
+            # of type Dict, OutcomeDict, str, list, HintDict and other types
```

Comment:
We should not be using Any. Wouldn't AcceptableChangeDictTypes or similar work?

Reply:
If we use `AcceptableChangeDictTypes or Union` then we have to use `assert isinstances` ( or some casts ) in the below places to narrow down the type of `new_value` from `Union` to a particular type.

Reply:
I'm not sure I follow -- wouldn't you need to do that as well if you use Any? If so, it seems that AcceptableChangeDictTypes or Union constrains it better.

Reply:
Any is removed!